episodes/2-identify-the-problem.Rmd

```diff
@@ -133,29 +122,25 @@ rodents <- rodents %>% filter(taxa == "Rodent")
 
 In general, when encountering a semantic error for which a remedy is not immediately apparent, some steps to take include:
 
-1. Reading any warning or informational messages that may pop up when executing your code.
-
-2. Changing the input to your function call to see if the behavior is ... 
+1.  Reading any warning or informational messages that may pop up when executing your code.
 
-2. Pulling up the R Documentation for that function (which may be specific to a package, such as with ggplot).
+2.  Changing the input to your function call to see if the behavior is ...
```

Comment:
What's the thought here?

Reply:
Hmmm I need to think more, it's not super fleshed out